public/scripts/main.js

```diff
@@ -88,108 +100,108 @@ function prevQuestion() {
+    $("#" + e.code.toLowerCase()).toggleClass("pressed");
   }
-  if(e.key.toLowerCase()=="alt" || e.key.toLowerCase()=="shift" || e.key.toLowerCase()=="meta"){
+  if (e.key.toLowerCase() == "alt" || e.key.toLowerCase() == "shift" || e.key.toLowerCase() == "meta") {
     let keyString = e.code;
-    if(e.code == FIREFOX_LEFT_COMMAND_STRING) {
+    if (e.code == FIREFOX_LEFT_COMMAND_STRING) {
       keyString = CHROME_LEFT_COMMAND_STRING
     } else if (e.code == FIREFOX_RIGHT_COMMAND_STRING) {
       keyString = CHROME_RIGHT_COMMAND_STRING
     }
-    $("#"+keyString.toLowerCase()).toggleClass("pressed");
+    $("#" + keyString.toLowerCase()).toggleClass("pressed");
   }
-  if(e.key.toLowerCase()=="capslock" && caps==false){
-    caps= true;
-    $("#"+e.key.toLowerCase()).toggleClass("pressed");
+  if (e.key.toLowerCase() == "capslock" && caps == false) {
+    caps = true;
+    $("#" + e.key.toLowerCase()).toggleClass("pressed");
     $('.letter').toggleClass('uppercase');
   }
-  else $("#"+e.key.toLowerCase() ).addClass("pressed");
+  else $("#" + e.key.toLowerCase()).addClass("pressed");
 }
 
 // Function called on KeyUp to reset the key by removing class = 'pressed'
 function release(e) {
-  if((e.which>=186 && e.which<=192)|| (e.which>=219 && e.which<=222)){
-    $("#"+e.code.toLowerCase()).removeClass("pressed");
+  if ((e.which >= 186 && e.which <= 192) || (e.which >= 219 && e.which <= 222)) {
+    $("#" + e.code.toLowerCase()).removeClass("pressed");
   }
-  if(e.key.toLowerCase()=="alt" || e.key.toLowerCase()=="shift" || e.key.toLowerCase()=="meta"){
+  if (e.key.toLowerCase() == "alt" || e.key.toLowerCase() == "shift" || e.key.toLowerCase() == "meta") {
     let keyString = e.code;
-    if(e.code == FIREFOX_LEFT_COMMAND_STRING) {
+    if (e.code == FIREFOX_LEFT_COMMAND_STRING) {
       keyString = CHROME_LEFT_COMMAND_STRING
     } else if (e.code == FIREFOX_RIGHT_COMMAND_STRING) {
       keyString = CHROME_RIGHT_COMMAND_STRING
     }
-    $("#"+keyString.toLowerCase()).removeClass("pressed");
+    $("#" + keyString.toLowerCase()).removeClass("pressed");
   }
-  if(e.code.toLowerCase()=="space"){
+  if (e.code.toLowerCase() == "space") {
     $("#space").removeClass("pressed");
   }
-  if(e.key.toLowerCase()=="capslock"){
-    $("#"+e.key.toLowerCase()).toggleClass("pressed");
+  if (e.key.toLowerCase() == "capslock") {
+    $("#" + e.key.toLowerCase()).toggleClass("pressed");
     $('.letter').toggleClass('uppercase');
-    caps=false;
-  } 
-  else{
-    $("#"+e.key.toLowerCase() ).removeClass("pressed");
+    caps = false;
+  }
+  else {
+    $("#" + e.key.toLowerCase()).removeClass("pressed");
   }
 }
 
 // May have to be removed. Not being used currently
-function highlightNextKey(params){
-  $("#"+nxt.toLowerCase()).toggleClass("pressed");
-  <!-- var params = { width:1680, height:1050 }; -->
-    <!-- var str = jQuery.param( params ); -->
-    <!-- $( "#results" ).text( str ); -->
```

Comment:
backend is broken. Fixed the comments

Reply:
Yes I'm surprised that was still in there. Thanks for removing it!